recipes/jarvis-tools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "jarvis-tools" %}
+{% set version = "2020.10.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jarvis-tools-{{ version }}.tar.gz
+  sha256: b4cc45267100fe2cc4d114d1453ad39e87d35244ce511c44b1fc99ca277f8a07
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```

Comment:
I am a bit surprised that there are no minimum versions here at all. I see that upstream does not have any version requirements in their setup.py either. Do you think that jarvis-tools actually work with any version of these dependencies?

Reply:
Determining the minimum versions for all the packages would be painstaking. I am involved with some other projects ([fipy](https://github.com/conda-forge/fipy-feedstock/blob/master/recipe/meta.yaml) and [pymks](https://github.com/conda-forge/pymks-feedstock/blob/master/recipe/meta.yaml) and we don't do that).